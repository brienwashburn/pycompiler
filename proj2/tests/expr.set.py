{a}
{a,}
{a, b, c, d}
{a, b, c, d,}
{1, 2, 3, 4, 5}
{x for x in range(10)}
