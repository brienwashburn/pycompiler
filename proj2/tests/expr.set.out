(Module
 (Expr (Set (Name a)))
 (Expr (Set (Name a)))
 (Expr (Set (Name a) (Name b) (Name c) (Name d)))
 (Expr (Set (Name a) (Name b) (Name c) (Name d)))
 (Expr (Set (Num 1) (Num 2) (Num 3) (Num 4) (Num 5)))
 (Expr
  (SetComp
   (Name x)
   (for
    (Name x)
    in
    (Call
     (func (Name range))
     (args (Num 10))
     (keywords)
     (starargs #f)
     (kwargs #f))
    if))))
